packages/react-data-grid/src/utils/domUtils.tsx

```diff
@@ -27,3 +28,14 @@ export default function getScrollbarSize(): number {
 
   return size;
 }
+
+export function isPositionStickySupported() {
```

Comment:
It's wasteful to not have this as a constant, or to not at least cache the result.
```ts
export const isPositionStickySupported = (() => {
  if (positionSticky === undefined) {
    const el = document.createElement('a');
    const mStyle = el.style;
    mStyle.cssText = 'position:-webkit-sticky;position:sticky';

     positionSticky = mStyle.position ? mStyle.position.includes('sticky') : false;
  }
  return positionSticky;
})();
```

Reply:
We are caching `positionSticky` and it is only checked once. Am I missing anything? Did you mean we should use a constant instead of a function?